src/encoded/static/libs/measurement.js

```diff
@@ -0,0 +1,28 @@
+
+export default class Measurement {
+    /**
+    * Takes a magnitude and unit (singular form) and returns the measurement
+    * with the proper unit in singular or plural, depending on magnitude.
+    * Invalid measurements are turned as- Unknown
+    * @param {number} magnitude - Measured value
+    * @param {string} unit - Measurement system in singular form
+    */
+    static format(magnitude, unit) {
+        // parseFloat used because parseInt only keeps integer
+        const magnitudeAsFloat = parseFloat(magnitude);
+
+        if (Number.isNaN(magnitudeAsFloat)) {
+            return 'Unknown';
+        }
+
+        if (magnitudeAsFloat === 0) {
+            return '0';
+        }
+
+        const measurement = `${magnitudeAsFloat} ${unit}`.trim();
+
+        return magnitudeAsFloat === 1
+          ? measurement
+          : `${measurement}s`;
+    }
+}
```

Comment:
What’s the advantage of a class with a single static method instead of a function without a class?

Reply:
Measurement seems like functionality we will expand on. That is why I made it a class. I made it a static because it is a utility, so does not depend on this. You got a valid point if you think it violates YAGNI and should be a function or utility-object with just a collection of function as these use less memory.

Reply:
I’m not going to gate approval on this; I just thought I would have difficulty explaining it to someone else. If you have a vision for how it might be expanded on in the future, I’ve no problem with it. I suppose even if it eventually got more static functions and no methods, there’s a name-space aspect to it (“This has to do with Measurement.”).